Parsers/ASimFileEvent/Parsers/ASimFileEvent.yaml

```diff
@@ -34,6 +34,7 @@ Parsers:
   - _ASim_FileEvent_MicrosoftWindowsEvent
   - _ASim_FileEvent_Native
   - _ASim_FileEvent_SentinelOne
+  - _ASim_FileEvent_VMwareCarbonBlackCloud
 ParserQuery: |
   let DisabledParsers=materialize(_GetWatchlist('ASimDisabledParsers') | where SearchKey in ('Any', 'ExcludeASimFile') | extend SourceSpecificParser=column_ifexists('SourceSpecificParser','') | distinct SourceSpecificParser| where isnotempty(SourceSpecificParser));
   let ASimBuiltInDisabled=toscalar('ExcludeASimFileEventBuiltIn' in (DisabledParsers) or 'Any' in (DisabledParsers));
```

Comment:
@jayeshprajapaticrest
please include VMWare carbon caller in ParserQuery property

Reply:
Done